doc/performance-testing/cloud-run/0001_stress-test-december-2021.md

```diff
@@ -0,0 +1,86 @@
+| Clients hatch rate   | 50 |
+| Wait time minimum | 1 |
+| Wait time maximum | 2 |
+| Requests JSON | test_benchmark_business.json |
+| Runtime | 10m |
+
+## Results
+
+| Clients per instance | Requests per second | 99th percentile Max CPU Usage (%) | 99th percentile response time (ms) | Error rate (%) | Output |
+| --- | --- | --- | --- | --- | --- |
+| 50  | 59.63 | 32.0  | 140 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T09:23:30) |
+| 60  | 71.52 | 37.0  | 140 | 0.006 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T13:39:53) |
+| 70  | 83.16 | 43.0  | 150 | 0.008 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T14:03:46) |
+| 80  | 94.70 | 50.0  | 150 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T14:23:23) |
+| 90  | 106.09 | 56.0 | 170 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T14:42:00) |
+| 100 | 117.17 | 61.0 | 180 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T14:57:02) |
+| 110 | 127.20 | 68.0 | 260 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T15:29:57) |
+| 120 | 135.18 | 75.0 | 390 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T15:49:24) |
+| 130 | 138.15 | 80.0 | 500 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-09T09:42:03) |
+| 140 | 140.66 | 81.0 | 630 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-09T10:14:49) |
+| 150 | 130.13 | 81.0 | 990 | 0.001 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-09T10:32:16) |
+| 160 | 135.59 | 99.0 | 740 | 0.005 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-09T10:52:38) |
+
+- The 99th percentile max cpu usage as reported in Grafana
+
+## Errors and Failures
+
+- 60 Clients per instance:
+    - 4 x "Expected a (200) but got a (0) back when getting page"
+- 70 Clients per instance:
+    - 4 x "Expected a (200) but got a (0) back when getting page"
+    - 1 x "Expected a (302) but got a (0) back when getting page"
+- 150 Clients per instance:
+  - 1 x "Expected a (302) but got a (0) back when getting page"
+- 160 Clients per instance:
+  - 3 x "Expected a (200) but got a (0) back when getting page"  
+
+## Observations
+
+- Some early errors that didn't persist when we increased the load.
```

Comment:
Not really sure what this means, sorry

Reply:
Updated this so hopefully it is more clear 👍